src/tests/routes/activities.test.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @jest-environment jsdom
+ */
```

Comment:
- [ ] Please 🙏🏽 remove commented code while it is not to help to understand your code, remember YAGNI.

Reply:
@franclobo, thanks once again  but this comment block is required by jest to know the environment we're running tests in. In this case, the DOM environment